### PR TITLE
review commit

### PR DESCRIPTION
**Pull request by MunteanuM**
Email registration new along the other trello tickets

### CODE REVIEW

**Comment by MunteanuM** [comment]
[[BE] Check email address before creating an account](https://trello.com/c/NNx2bWZi/10-be-check-email-address-before-creating-an-account)

**Comment by maxinasa** [comment]
Please remove the .pyc files from the repo and add them in the gitignore file.

File: app_users/forms.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    class Meta:
+        model = User
+        fields = [
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+            'password1',
+            'password2',
+            ]
```
Comment (maxinasa) [inline app_users/forms.py:20]: Please, use the pyflake or pylint to check your code before committing. For example, here, there is no newline at the end of the file.

File: app_users/token.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+from django.utils import six
+
+class AccountActivationTokenGenerator(PasswordResetTokenGenerator):
+    def _make_hash_value(self, user, timestamp):
+        return (
+            six.text_type(user.pk) + six.text_type(timestamp) +
+            six.text_type(user.profile.email_confirmed)
+        )
+
+account_activation_token = AccountActivationTokenGenerator()
```
Comment (maxinasa) [inline app_users/token.py:11]: One of these two files is a duplicate, right?

File: app_users/urls.py [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
 
 from . import views
 
 urlpatterns = [
     path('login_user/', views.login_user, name="login_user"),
-    path('logout_user',views.logout_user, name="logout_user"),
+    path('logout_user/',views.logout_user, name="logout_user"),
+   # path('register_user/', views.register_user, name="register_user"),
+    path('signup/', SignUpView.as_view(), name='signup'),
+    path('activate/<uidb64>/<token>/', ActivateAccount.as_view(), name='activate'),
```
Comment (maxinasa) [inline app_users/urls.py:12]: Although it's easier to search and match, it's recommended to use hyphens in URIs.

**Review by maxinasa — APPROVED** [review]

File: basepage/templates/basepage/base.html [hunk 4]
```diff
@@ -43,6 +43,7 @@ <h1>
 
     {%else%}
     <a href="{% url 'login_user' %}">login</a>
+    <a href="{% url 'signup' %}">register</a>
```
Comment (drejleeo) [inline basepage/templates/basepage/base.html:46]: On the other PR I said something about using relative path. I meant this. This is the django way of referencing internal paths. These tags `{% something %}` are called template tags. The url template tag solves url referenced by their path names using a reverse. You can even configure your own template tags. You can give it a read on the documentation if you are curious.

**Review by drejleeo — CHANGES_REQUESTED** [review]
Again, remove `.pyc` files

**Pull request closed without merging**